Dockerfile

```diff
@@ -13,6 +13,12 @@ RUN sed -i.bak "s/..\/..\/..\/nanshe_workflow/../g" /nanshe_workflow/.git/config
 ADD entrypoint.sh /usr/share/docker/entrypoint_2.sh
 
 RUN for PYTHON_VERSION in 2 3; do \
+        # Pin `jupyter_client` to workaround an issue with `pytest`.
+        # Please see the linked PR.
+        #
+        # https://github.com/conda-forge/jupyter_client-feedstock/pull/14
+        #
+        echo "jupyter_client 5.1.0" >> "/opt/conda${PYTHON_VERSION}/conda-meta/pinned" && \
```

Comment:
Moved to `nanshe/nanshe_notebook` with PR ( https://github.com/nanshe-org/docker_nanshe_notebook/pull/5 ). Downgrading it here seemed to be problematic anyways. Not to mention the workaround would be best in `nanshe/nanshe_notebook` anyways as that is where we install `jupyter_client` initially.